libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -437,6 +437,49 @@ protected async Task<DialogTurnResult> OnEndOfActionsAsync(SequenceContext seque
         protected async Task<RecognizerResult> OnRecognize(SequenceContext sequenceContext, CancellationToken cancellationToken = default(CancellationToken))
         {
             var context = sequenceContext.Context;
+            var noneIntent = new RecognizerResult()
+            {
+                Text = context.Activity.Text ?? string.Empty,
+                Intents = new Dictionary<string, IntentScore>()
```

Comment:
Not for now, but we should have a static factory method in the IntentScore class, called none, so you do IntentScore.None and it works like string.Empty. I've seen thiscode in a few places already so it could probably be useful. I can add it when I am working around this code.